engine/src/main/java/org/terasology/world/internal/WorldProviderWrapper.java

```diff
@@ -22,12 +22,14 @@
 import org.terasology.world.WorldChangeListener;
 import org.terasology.world.WorldProvider;
 import org.terasology.world.block.Block;
+import org.terasology.world.chunks.CoreChunk;
```

Comment:
We don't need to make any changes to this file in the end. Could you revert whatever changes happened here (formatting etc.) to keep the git diff cleaner?